devito/ir/iet/nodes.py

```diff
@@ -366,7 +366,7 @@ class Iteration(Node):
         The for-loop direction. Accepted:
         - ``Forward``: i += stepping (defaults)
         - ``Backward``: i -= stepping
-    properties : IterationProperty or list of IterationProperty, optional
+    properties : Property or list of Property, optional
```

Comment:
I get the idea, but "Property" feels very vague/general, maybe "CompilerProperty" or "PasssProperty" or just keep the previous IterationProperty

Reply:
Would vote for IterationProperty

Reply:
It is **not** an IterationProperty (anymore), otherwise I'd have kept the original name